tgui/bin/tgui

```diff
@@ -140,16 +145,6 @@ if [[ ${1} == "--install-git-hooks" ]]; then
   exit 0
 fi
 
-## Continuous integration scenario
-if [[ ${1} == "--ci" ]]; then
-  task-clean
-  task-install
-  task-eslint
-  task-webpack --mode=production
-  task-validate-build
-  exit 0
-fi
-
```

Comment:
This along with the change to build_tgui.sh seems to have disabled the github CI check for unbuilt bundle.